bridge/elrond/client_test.go

```diff
@@ -25,6 +30,70 @@ func (e TransactionError) Error() string {
 	return string(e)
 }
 
+func createMockArguments() ClientArgs {
+	return ClientArgs{
+		Config: bridge.Config{
+			BridgeAddress:        "bridge address",
+			PrivateKey:           "grace.pem",
+			NonceUpdateInSeconds: 1,
+		},
+		Proxy: &mock.ElrondProxyStub{},
+	}
+}
+
+func TestNewClient(t *testing.T) {
+	t.Parallel()
+
+	args := createMockArguments()
```

Comment:
this could have been a test suite so all constructor tests will run a grouped manner

Reply:
done